### PR TITLE
feat: release a2 english (first three blocks)

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

This assumes that we're going to release the first 6 blocks. If that's not the case, we `intro.json` will need adjusting with the correct number and the `showUpcoming` flags will need flipping.

/learn :

image

/learn/a2-english-for-developers:
image

Edit: it's says 3 courses now, not 6.

### CODE REVIEW

**Comment by naomi-lgbt** [comment]
Do we not want to release this as a `new` certification (so it can be gated to only show up on the English site)?

**Comment by moT01** [comment]
I don't think we're going the `new` route this time. Like Quincy mentioned - I think part of why we did that before was cause it was RWD (the first superblock) - these will be further down

**Comment by ojeytonwilliams** [comment]
I don't think it can be done without a re-write to superblock order and it's too close to release to mess with, imo.

**Review by naomi-lgbt — APPROVED** [review]
Okie dokie, we should be good to land this then

**Review by ShaunSHamilton — APPROVED** [review]
Fewer tests == more better

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -1197,7 +1197,7 @@
     "title": "A2 English for Developers (Beta)",
     "intro": [
       "This course teaches the English language, designed for developers.",
-      "More text here."
+      "In this beta version of the curriculum you can try out the first 3 of the 18 courses we are developing."
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:1200]:
```suggestion
      "In this beta version of the curriculum you can try out the first 2 of the 18 courses we are developing."
```

**Pull request merged**